### DISCUSSION TITLE
How convert into JSON object std:string varabile

### DISCUSSION THREAD

**Discussion by yusrub** [nlohmann/json#3024] [Q&A]
```
std:string arg = "{\"hello\" : 1}";
```
I want to convert arg into JSON object, I tried different ways but I did not work for mw
```json j3 = arg_json;```
OR 
```json j3 = json::parse(arg);```
How i can do?

**Comment by nlohmann** [comment]
This is done with `parse`. If that did not work - what was the error message?

In order to asses your issue, we need the following information:

- What is the issue you have?

- Please describe the steps to reproduce the issue. Can you provide a small but working code example?

- What is the expected behavior?

- And what is the actual behavior instead?

- Which compiler and operating system are you using? Is it a [supported compiler](https://github.com/nlohmann/json#supported-compilers)?

- Did you use a released version of the library or the version from the `develop` branch?

- If you experience a compilation error: can you [compile and run the unit tests](https://github.com/nlohmann/json#execute-unit-tests)?
